rokka/client.go

```diff
@@ -13,16 +15,25 @@ type Client struct {
 type Config struct {
 	APIAddress string
 	APIVersion string
-	APIKey string
+	APIKey     string
 	HTTPClient *http.Client
 }
 
+type StatusCodeError struct {
+	StatusCode int
+	Body       []byte
```

Comment:
you could also leave it as `io.ReadCloser` here and only read all in case you need it. But I guess that's rather a matter of preference.

Reply:
Not possible, because we close the body.